spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -92,16 +92,6 @@ describe("RustCrypto", () => {
             const res = await rustCrypto.preprocessToDeviceMessages(inputs);
             expect(res).toEqual(inputs);
         });
-
-        it("should pass through one time key counts", async () => {
-            const oneTimeKeyCounts = new Map<string, number>([["signed_curve25519", 50]]);
-            await expect(rustCrypto.preprocessOneTimeKeyCounts(oneTimeKeyCounts)).resolves.not.toBeDefined();
-        });
-
-        it("should pass through unused fallback keys", async () => {
-            const unusedFallbackKeys = new Set(["signed_curve25519"]);
-            await expect(rustCrypto.preprocessUnusedFallbackKeys(unusedFallbackKeys)).resolves.not.toBeDefined();
-        });
```

Comment:
would be nice to add a test for `processKeyCounts`.

Reply:
Previous I added tests and I was told that they weren't pertinent here and add integration test instead 😞

Reply:
oh, sorry. That's probably fair.